astropy/units/quantity_helper/helpers.py

```diff
@@ -34,7 +35,28 @@ def get_converter(from_unit, to_unit):
     """Like Unit.get_converter, except returns None if no scaling is needed,
     i.e., if the inferred scale is unity.
     """
-    converter = from_unit.get_converter(to_unit)
+    try:
+        converter = from_unit.get_converter(to_unit)
+    except AttributeError as e:
+        # Check for lack of unit only now, to avoid delay for cases where a unit
```

Comment:
Double unit here (last word in this line, first in the next)

Reply:
Oops, another typo, thanks for catching it... Fix in #16881